react/tic-tac-toe/src/scss/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$button-transition-timing: ease;
+$default-transition-duration: 100ms;
```

Comment:
I suggest you take a look at [these articles](https://material.io/design/motion/speed.html#) they have very good explanations on how and when to use animations. Not that I want you to change it, but would be cool to check it out.